### ISSUE TITLE
New Focus not reported by screen reader for bar jumps with Ctrl F

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#32516]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

The screen reader is silent after using Ctrl F, the new focused position or note is not read and confusion as to whether it has already jumped leads screen reader users to press Esc to close the find window jumping them back to the start of the score.  In contrast if you enter a rehearsal mark it announces this when focus moves to it.  Also, if you pause playback and jump to that position with Ctrl space it announces the newly focused note.

STEPS:
1.  On any score (with NVDA on) press Ctrl F and type a bar number, say 12, press enter.
2. Notice that the focus has moved as expected to bar 12, but the screen reader has not announced anything.
3. Press Esc, this is what many users will automatically do thinking because the screen reader has not spoken that they need to shut the pop up.
4. On a score with a rehearsal mark press Ctrl F, type the rehearsal mark and press enter.
5. Notice that the score jumps to the rehearsal mark as expected and this time the screen reader announces it immediately when focused.
6. Press space to begin playback and a few bars later press Ctrl space.
7. Notice that the focus does jump to the last played note and the screen reader announces the new note/position.

It would be ideal if the screen reader announced the new position for bar jumps as described in the first two steps.  But if that is not possible, then simply announcing the usual thread of note pitch/duration, bar/beat, etc when it is focused would be sufficient.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.7 beta

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by bkunda** [comment]
@shoogle could you pls regression check this one?
